plugins/env/README.md

```diff
@@ -1,4 +1,39 @@
+## Env Plugin
 Plugin to manage ENV settings for projects and write .env files during deploy.
 
 Includes `/projects/:permalink/environment?deploy_group=permalink` endpoint that returns the .env content
 for a project and deploy_group.
+
+## GitHub to manage environment variables
```

Comment:
maybe `Github` -> `Git repo` so it sounds more universal

Reply:
It does but right now it only works with GitHub

Reply:
We may want to state the caveat that this currently only works with GitHub.   We may want to make the GITHUB interface object more generic.   But that is another PR